src/LanguageHelper.php

```diff
@@ -33,6 +35,52 @@ public function exists($lang, $basePath)
 	}
 
 	/**
+	 * Searches for a specific localise file for a given language. Falls back to the .
+	 *
+	 * @param   string  $lang      Language to check.
+	 * @param   string  $basePath  Base path to the language folder.
+	 *
+	 * @return  boolean  True if the language exists.
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function findLocalise($lang, $basePath)
+	{
+		// Look for a language specific localise class
+		$class = str_replace('-', '_', $lang . 'Localise');
+		$paths = array();
+
+		$paths[0] = $basePath . "/overrides/$lang.localise.php";
+		$paths[1] = $basePath . "/$lang/$lang.localise.php";
+
+		ksort($paths);
+		$path = reset($paths);
+
+		while (!class_exists($class) && $path)
+		{
+			if (file_exists($path))
+			{
+				require_once $path;
+			}
+
+			$path = next($paths);
+		}
+
+		// If we have found a match initialise it and return it
+		if (class_exists($class))
+		{
+			$localiseClass = new $class;
+
+			return $localiseClass;
+		}
+
+		// Return the default class if no specific instance is found
+		$localiseClass = new AbstractLocalise;
```

Comment:
If you're calling it an abstract class in the class name, then it should be abstract or the class should be renamed.